bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html

```diff
@@ -99,21 +96,13 @@ <h3 id="Level1a">{{ _('Level 1a - Named Voucher') }}</h3>
 <table class="table module-table">
   <tr>
     <td>mozilla.org/trunk</td>
-    <td>David Boswell or Reed Loden</td>
+    <td>Reed Loden</td>
   </tr>
   <tr>
     <td>mozilla.com/tags/production</td>
     <td>John Slater</td>
```

Comment:
Not really a code comment, I just happened to notice this: John Slater is no longer with Mozilla, and I'm not sure who would be the current voucher for this repository. But we're also trying to stop using it as we move everything to git/bedrock. Maybe just delete the whole entry?